plugins/modules/swim_intent.py

```diff
@@ -982,15 +996,15 @@ def get_diff_tagging(self):
                 device_family_identifier=self.have.get("device_family_identifier"),
                 device_role=tagging_details.get("device_role")
             )
-            self.log("Image params for un-tagging image as golden:" + str(image_params))
+            self.log("Image params for tagging image as golden: {0}".format(str(image_params)), "INFO")
```

Comment:
self.log("Parameters for tagging the image as golden: {0}".format(str(image_params)), "INFO")

Reply:
updated.